Softeq.XToolkit.WhiteLabel/Mvvm/RootFrameNavigationPageViewModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using Softeq.XToolkit.WhiteLabel.Navigation;
+
+namespace Softeq.XToolkit.WhiteLabel.Mvvm
+{
+    public class RootFrameNavigationPageViewModel<T> : RootFrameNavigationViewModelBase where T : IViewModelBase
```

Comment:
Navigation behavior looks the same as `DrillRootPageViewModel` (navigation stack for each root)

Reply:
Sure. Behavior is the same - we provide navigation page with first page in stack. The difference is that in case of DrillRootPageViewModel if you need 5 stacks (say 5 elements in your side/tab menu) you have to create 5 such root view models in your project that do exactly the same, only set different first pages (in addition to your 5 first screen view models). In case of RootFrameNavigationPageViewModel<T> you do not need to create these additional view models and navigation pages and you can use this generic class that performs the same logic.

Reply:
Agree!